daemon/GraknDaemon.java

```diff
@@ -244,5 +261,60 @@ private void clean() {
         storageExecutor.clean();
         serverExecutor.clean();
     }
+
+    private static final int SLOW_ANIM_RATE = 5000;
+    private static final int FAST_ANIM_RATE = 100;
+
+    private class MigrationArgs {
+        private List<String> remaining;
+        private boolean fastAnim = true;
+
+        private MigrationArgs(List<String> args) {
+            remaining = new ArrayList<>();
+            for (String arg : args) {
+                if (!arg.startsWith("--")) {
+                    remaining.add(arg);
+                }
+
+                if (arg.equals("--no-anim")) {
+                    fastAnim = false;
+                }
+            }
+
+            if (remaining.size() != 2) {
+                throw new IllegalArgumentException("Must use exactly 2 arguments: <keyspace> <file>");
+            }
+        }
+    }
+
+    private void export(List<String> args) {
+        MigrationArgs margs = new MigrationArgs(args);
+
+        try (MigrationClient client = new MigrationClient();
+             ProgressPrinter printer = new ProgressPrinter("export", margs.fastAnim ? FAST_ANIM_RATE : SLOW_ANIM_RATE)) {
+            client.export(margs.remaining.get(0), margs.remaining.get(1), printer);
+        } catch (Exception exception) {
+            throw new RuntimeException(exception);
+        }
+    }
+
+    private void import_(List<String> args) {
+        MigrationArgs margs = new MigrationArgs(args);
+
+        try (MigrationClient client = new MigrationClient();
+             ProgressPrinter printer = new ProgressPrinter("import", margs.fastAnim ? FAST_ANIM_RATE : SLOW_ANIM_RATE)) {
+            client.import_(margs.remaining.get(0), margs.remaining.get(1), printer);
+        } catch (Exception exception) {
+            throw new RuntimeException(exception);
+        }
+    }
+
+    private void exportSchema(List<String> args) {
+        try (MigrationClient client = new MigrationClient()) {
+            System.out.println(client.exportSchema(args.get(0)));
```

Comment:
It is the keyspace. I tried to keep the argument parsing code as simple as possible since we'll move to Picocli soon, but I'm thinking now that it might be better to do a full Apache CLI parse anyway.